firstvoices/backend/models/characters.py

```diff
@@ -256,12 +266,18 @@ def preprocess_transducer(self):
         """
         if self.input_to_canonical_map:
             preprocess_settings = self.default_g2p_config["preprocess_config"]
-
+            # need to query for self in order to correctly grab property as dict
+            self_with_object = Alphabet.objects.only("input_to_canonical_map").get(
+                id=self.id
+            )
```

Comment:
This is a fascinating fix, wow. So the tests were having trouble handling the json correctly in some operations? If possible, it would be much better to put this fix in the test itself since it really seems like a limitation of the test framework. (Also probably worth logging a bug to the test framework project.)

Reply:
It doesn't look like it works to just put the query in the test. Seems like the test objects behave slightly differently wrt this function than actual objects do in practice. I will try to come up with a MWE and send it to the relevant test project (not sure which one exactly it comes from) -- should I keep this weird fix in, or should I go back to skipping the test?

Reply:
Since this adds an extra database hit in a function that we will likely be running in bulk, I think we should revert to skipping that test. Maybe we need to start a list of required manual tests?